webview.h

```diff
@@ -1290,9 +1290,8 @@ inline std::wstring find_edge_webview_client_dll(unsigned int min_api_version) {
   client_dll_path += L"x64";
 #elif defined(_M_X86)
   client_dll_path += L"x86";
-#elif defined(_M_ARM64)
-  // TODO: Check if this is actually possible
-  client_dll_path += L"arm64";
```

Comment:
Building is possible with LLVM MinGW, and the resulting binary runs. So there isn't actually any reason to remove this.

Reply:
@janh I removed this particular code because I had no way to validate it. I would like to make sure that it works before merging it.

Reply:
I have finally been able to validate this code on ARM64 hardware.